templates/Dockerfile-deb.tmpl

```diff
@@ -71,6 +71,7 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main' >
         liblz4-1 \
         libzstd1 \
         libzstd-dev \
+        sudo \
```

Comment:
It is not usually advised to use sudo in docker environment. Why is this addition necessary?

Did you consider `gosu` or `USER` docker directives instead?

Reply:
Actually OneBranch needs it. They use sudo 
https://msdata.visualstudio.com/Database%20Systems/_build/results?buildId=68259209&view=logs&j=570eab73-0366-550a-dd5d-1b82c1121f48&t=49b1901d-5c98-4bfe-afc8-a8deb533389f&l=98

Reply:
I guess it is ok then. We can not do anything about this if OneBranch decides to use an antipattern 😮‍💨